python-test/features/steps/functions.py

```diff
@@ -0,0 +1,77 @@
+import requests
+import json
+# from pygments import highlight, lexers, formatters
+import os
+import docker
+import re
+
+base_orb_url = os.getenv('ORB_URL', "https://beta.orb.live")
```

Comment:
why do we need two variables, `ORB_ADDRESS` and `ORB_URL`?
We could have a single `ORB_ADDRESS` and do
```python
base_orb_url = "https://" + os.getenv('ORB_ADDRESS', "beta.orb.live")

Reply:
agreed we only need one variable here. but note that this variable is currently processed by the docker entry script, not orb-agent directly

https://github.com/ns1labs/orb/blob/develop/agent/docker/orb-agent-entry.sh#L10